mycore-base/src/main/resources/xsl/functions/mcrversion.xsl

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="3.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:mcrversion="http://www.mycore.de/xslt/mcrversion"
+                exclude-result-prefixes="fn xs">
+
+    <xsl:function name="mcrversion:getCompleteVersion" as="xs:string">
```

Comment:
`mcrversion:completeVersionString`?

Reply:
i renamed it to mcrversion:get-complete-version. Its the same method names as in MCRCoreVersion just with xslt function naming convetions